psi4/src/psi4/optking/set_params.cc

```diff
@@ -314,7 +314,15 @@ void set_params(void)
       Opt_params.conv_max_disp  = 6.0e-5;  Opt_params.i_max_disp = true;
       Opt_params.conv_rms_disp  = 4.0e-5;  Opt_params.i_rms_disp = true;
     }
-    else if (Opt_params.general_conv == "GAU_VERYTIGHT") {
+    else if (Opt_params.general_conv == "INTERFRAG_GAU_TIGHT") {
+      Opt_params.i_untampered = true;
+      Opt_params.conv_max_DE    = 1.0e-6;  Opt_params.i_max_DE = true;
+      Opt_params.conv_max_force = 1.5e-5;  Opt_params.i_max_force = true;
+      Opt_params.conv_rms_force = 1.0e-5;  Opt_params.i_rms_force = true;
+      Opt_params.conv_max_disp  = 6.0e-4;  Opt_params.i_max_disp = true;
+      Opt_params.conv_rms_disp  = 4.0e-4;  Opt_params.i_rms_disp = true;
+    }
+if (Opt_params.general_conv == "GAU_VERYTIGHT") {
```

Comment:
can add to docs table (https://github.com/psi4/psi4/blob/master/doc/sphinxman/source/optking.rst) while you're at it.